lib/rucio/core/did.py

```diff
@@ -1285,12 +1286,17 @@ def get_did_meta(scope, name, session=None):
     :param scope: the scope of did
     :param name: the name of the did
     """
-    if session.bind.dialect.name == 'oracle' or session.bind.dialect.name == 'sqlite':
+    if session.bind.dialect.name == 'sqlite':
         raise NotImplementedError
+    if session.bind.dialect.name == 'oracle':
```

Comment:
Just an idea for later: why might think about doing these db compatibility checks via a function decorator?